app/src/main/java/ch/epfl/cs311/wanderwave/ui/components/player/ExclusivePlayer.kt

```diff
@@ -49,103 +55,85 @@ fun ExclusivePlayer(
       horizontalAlignment = Alignment.CenterHorizontally,
       verticalArrangement = Arrangement.SpaceBetween) {
         Column(verticalArrangement = Arrangement.Top) {
-          PlayerDragHandleComponent(checked)
-          Box(
-              contentAlignment = Alignment.Center,
-              modifier = Modifier.fillMaxWidth().padding(top = 10.dp)) {
-                SwitchComponent(checked)
-              }
-          Spacer(modifier = Modifier.height(10.dp))
-          PlayerIconButtonRowComponent()
-          Spacer(modifier = Modifier.height(15.dp))
-          VotingButtonsComponent(selectedVote)
+          PlayerDragHandle(duration1 = 1500, duration2 = 1500, duration3 = 1500, startColor = pink)
         }
-        TrackInfoComponent(uiState)
-        SliderComponent(progress)
-        PlayerControlRowComponent(viewModel, uiState)
+        Column(
+            verticalArrangement = Arrangement.Bottom,
+            horizontalAlignment = Alignment.CenterHorizontally) {
+              TrackImageComponent(uiState = uiState, fetchAlbumImage = viewModel::fetchImage)
```

Comment:
A little spacer here would be nice

Reply:
Because it's space between Vertical arrangement I think it's fine no ?